### PR TITLE
ENDOC-107 Added openshift landing page

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
This is visible on https://es-entando.github.io/openshift.html currently. This shouldn't be merged/published until we get the corresponding entando-releases change in place for the okd files.

**Comment by joewhite101** [comment]
- Image is pretty blury on my screen

**Comment by nshaw** [comment]
> * Image is pretty blury on my screen

Agreed. My original CRC screenshot was a bit better but I'll check with Alessio on it.

**Review by joewhite101 — APPROVED** [review]
LGTM

**Comment by joewhite101** [comment]
LGTM

File: vuepress/docs/.vuepress/components/SpecialLayout.vue [hunk 1]
```diff
@@ -75,7 +75,119 @@
...
+              <div class="instruction">
+                oc new-project entando
+              </div>
+              <p>Download Helm chart. Note: if you have OpenShift 3.11, use entando-okd3.yaml for the remaining steps.</p>
+              <div class="instruction">
+                curl -L -C - -O <span>\<br></span>https://raw.githubusercontent.com/entando/entando-releases/v6.2.0/dist/qs/entando-okd4.yaml
+              </div>
+              <p>Set an environment variable using the IP address from Step 1. For a managed cluster address, you can remove .nip.io from the value.</p>
+              <div class="instruction">
+                IP=MY_OPENSHIFT_IP.nip.io
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/.vuepress/components/SpecialLayout.vue:153]: Agreed, tweaked the language slightly but may need to revisit later.

**Pull request merged**